repeater/src/main/scala/com.snowplowanalytics.snowplow.storage.bigquery.repeater/Recover.scala

```diff
@@ -0,0 +1,65 @@
+package com.snowplowanalytics.snowplow.storage.bigquery.repeater
+
+import fs2.Stream
+import cats.effect.Concurrent
+import cats.effect.Timer
+import blobstore.Path
+import blobstore.implicits.GetOps
+import io.circe.JsonObject
+import java.time.Instant
+import java.util.UUID
+import io.circe.generic.auto._, io.circe.parser._
+import cats.effect.Sync
+import cats.syntax.all._
+
+object Recover {
+
+  //TODO: hardcode proper GCS path
+  val DeadQueuePath = Path("gs://sp-storage-loader-failed-inserts-dev1-com_snplow_eng_gcp/dead_queue")
```

Comment:
Should not appear on a public repo like this.

Reply:
Good point - thanks. Do you have an idea where to put it for such one off job? env var feels right, but I would prefer to not touch TF for this task.

Reply:
https://github.com/snowplow-incubator/snowplow-bigquery-loader/blob/master/repeater/src/main/scala/com.snowplowanalytics.snowplow.storage.bigquery.repeater/RepeaterCli.scala#L36

Reply:
Thanks Dilyan. This means changes in TF unfortuntelly.

Reply:
Sorry, not sure I follow. Why does it mean changes in TF? Isn't it the same bucket that is being passed in as argument to Repeater on deploy?

Reply:
Well there are only 2 possibilities for configuration:
- either it's hard-coded
- or it needs to be provided, meaning TF

Reply:
(Dilyan's answer appeared while I was writing)

Reply:
Thanks @dilyand You might be right we already have it! Awesome - thanks!

Reply:
Forced pushed to remove the hardcoded path.